clang/lib/AST/Interp/ByteCodeExprGen.cpp

```diff
@@ -1596,6 +1599,9 @@ bool ByteCodeExprGen<Emitter>::visitZeroInitializer(QualType QT,
     return this->emitZeroSint64(E);
   case PT_Uint64:
     return this->emitZeroUint64(E);
+  case PT_IntAP:
+  case PT_IntAPS:
+    assert(false);
```

Comment:
Should this be an `llvm_unreachable`?

Reply:
All the `assert(false)` in this patch are just placeholders for things I need to implement in later patches. I'm trying to keep the patches smaller so they are easier to review.